lib/repl.js

```diff
@@ -1126,7 +1126,7 @@ function isRecoverableError(e, self) {
       self._inTemplateLiteral = true;
       return true;
     }
-    return /^(Unexpected end of input|Unexpected token)/.test(message);
+    return /^(Unexpected end of input|Unexpected token|missing \) after argument list)/.test(message);
```

Comment:
@thefourtheye  hrngh, I thought about `new RegExp`, but every time I do something like that some paranoid part of me worries about escaping.

Multiple regexps (or the array one I posted above) is a bit more verbose, but may be better in that regard.